addons/mass_mailing/models/mass_mailing.py

```diff
@@ -316,6 +316,18 @@ def _get_source_id(self, cr, uid, context=None):
     def _get_medium_id(self, cr, uid, context=None):
         return self.pool['ir.model.data'].xmlid_to_res_id(cr, uid, 'utm.utm_medium_email')
 
+    def create(self, cr, uid, vals, context=None):
+        if not vals['campaign_id']:
```

Comment:
As mass_mailing.campaign inheritS from utm.mixin, you should not create the utm campaign manually. However, as the name field is defined on mail.campaign, maybe you will have to write the name field on the utm.campaign ... Bref, seems louche. To check.